lib/catalogx_client/base_client.rb

```diff
@@ -60,8 +69,8 @@ def handle_request(url, http_verb, query_params: nil, body: nil)
       end
     end
 
-    def handle_result(result)
-      if SUCCESS_CODES.include?(result.status)
+    def self.handle_result(result)
+      if HTTPStatusCodes.success_codes.include?(result.status)
```

Comment:
Alternately, if you just want to check for 2xx error codes, you can also do `result.success?`
https://www.rubydoc.info/gems/faraday/Faraday/Response#success%3F-instance_method

Reply:
nice, I like this better. I'll change it